drivers/base/bootvid/common.c

```diff
@@ -15,8 +15,42 @@ ULONG VidpScrollRegion[4] =
     SCREEN_HEIGHT - 1
 };
 
+/*
+ * Boot video driver default palette is similar to the standard 16-color
+ * CGA palette, but it has Red and Blue channels swapped, and also dark
+ * and light gray colors swapped.
+ */
+const RGBQUAD DefaultPalette[BV_MAX_COLORS] =
+{
+    RGB(  0,   0,   0), /* Black */
+    RGB(128,   0,   0), /* Red */
+    RGB(  0, 128,   0), /* Green */
+    RGB(128, 128,   0), /* Brown */
+    RGB(  0,   0, 128), /* Blue */
+    RGB(128,   0, 128), /* Magenta */
+    RGB(  0, 128, 128), /* Cyan */
+    RGB(128, 128, 128), /* Dark Gray */
+    RGB(192, 192, 192), /* Light Gray */
+    RGB(255,   0,   0), /* Light Red */
+    RGB(  0, 255,   0), /* Light Green */
+    RGB(255, 255,   0), /* Yellow */
+    RGB(  0,   0, 255), /* Light Blue */
+    RGB(255,   0, 255), /* Light Magenta */
+    RGB(  0, 255, 255), /* Light Cyan */
+    RGB(255, 255, 255), /* White */
+};
+
+static BOOLEAN ClearRow = FALSE;
+
 /* PRIVATE FUNCTIONS **********************************************************/
 
+VOID
+NTAPI
+InitializePalette(VOID)
```

Comment:
At this point you could transform this function into a macro, or directly call InitPaletteWithTable with the default parameters, where needed.